File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        List<Integer> dice = new ArrayList<>();
        dice.add(sc.nextInt());
        dice.add(sc.nextInt());
        dice.add(sc.nextInt());
        Collections.sort(dice);

        Set<Integer> setDice = new HashSet<>(dice);
        if (setDice.size() == 1) {
            System.out.println(10000 + (dice.get(0) * 1000));
        } else if (setDice.size() == 2) {
            System.out.println(1000 + (dice.get(1) * 100));
        } else {
            System.out.println(100 * dice.get(2));
        }
    }
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
dice = list(map(int, input().split()))
dice.sort()

if dice[0] == dice[1] and dice[0] == dice[2] and dice[1] == dice[2]:
    print(10000 + (dice[0] * 1000))
elif dice[0] != dice[1] and dice[0] != dice[2] and dice[1] != dice[2]:
    print(max(dice) * 100)
else:
    if dice[0] == dice[1]:
        print(1000 + (100 * dice[0]))
    elif dice[0] == dice[2]:
        print(1000 + (100 * dice[0]))
    else:
        print(1000 + (100 * dice[1]))